File: app/src/main/java/com/example/poo_chatbot/Models/ChatModel.java
package com.example.poo_chatbot.Models;

public class ChatModel {
    public String messege;
    public boolean isSend;

    public ChatModel(String messege, boolean isSend){
        this.messege = messege;
        this.isSend  = isSend;
    }

    public ChatModel(){
    }

    public String getMessege() {
        return messege;
    }

    public void setMessege(String messege) {
        this.messege = messege;
    }

    public boolean isSend() {
        return isSend;
    }

    public void setSend(boolean send) {
        isSend = send;
    }
}

File: app/src/main/java/com/example/poo_chatbot/Models/ChatModel.kt
package com.example.poo_chatbot.Models

import android.R.id.message
import android.R.id.message





class ChatModel {
    var messege: String
    var isSend: Boolean = false

    constructor(messege: String, isSend: Boolean) {
        this.messege = messege
        this.isSend = isSend
    }
    fun setMessage(message: String) {
        this.messege = message
    }
    fun getMessage(): String {
        return messege
    }

}